charts/super-agent-deployment/values.yaml

```diff
@@ -116,6 +128,12 @@ config:
       #    endpoint: https://opamp.service.newrelic.com/v1/opamp
       #    headers:
       #      api-key: LICENSE_KEY
+      #    auth_config:
+      #      token_url: "http://fake.com/oauth2/v1/token"
```

Comment:
should nrStaging config change this? I wonder if we should add it "magically" (dame for endpoint few lines above)